### PR TITLE
fix: resolves issue with small video thumbnails in helper content

### PR DESCRIPTION
**Pull request by plittlewood-rpt**
# Pull request checklist

**Detail as per issue below (required):**

Resolves issue with tiny video thumbnail by forcing component to fill the available space. Issue was due to the svg element having a height of 1em when rendered. I'm not entirely sure where this was coming from, but the PR resolves the issue as shown below. The PR also alters one of the vite server properties which I had to adjust to get the application to run locally on my windows machine (using Ubuntu WSL). Without knowing the ecosystem very well I don't know if there'll be any adverse effects to changing this (I suspect not, given developers on Macs use localhost anyway), but please consider this change when reviewing.

image

fixes: #10942

### CODE REVIEW

**Review by RWilcox-Reapit — APPROVED** [review]

**Comment by plittlewood-rpt** [comment]
@willmcvay @bashleigh would one of you mind just giving me the nod with this - I feel like this might actually be a problem somewhere in Elements. This 'fixes' the issue as far as the Dev Portal goes, but feel like there might be more to it

File: packages/ts-scripts/src/vite/base-vite.js [hunk 1]
```diff
@@ -92,7 +92,7 @@ module.exports = (config, appName) =>
       alias: [{ find: '@', replacement: `${path.resolve(process.cwd(), 'src')}/` }],
     },
     server: {
-      host: true,
+      host: 'localhost',
```
Comment (bashleigh) [inline packages/ts-scripts/src/vite/base-vite.js:95]: Not sure about this change? Did you have an issue with the config?

**Review by bashleigh — COMMENTED** [review]
Have added a comment for now

**Comment by willmcvay** [comment]
> @willmcvay @bashleigh would one of you mind just giving me the nod with this - I feel like this might actually be a problem somewhere in Elements. This 'fixes' the issue as far as the Dev Portal goes, but feel like there might be more to it

@plittlewood-rpt this will fix it - it's a custom SVG only used on that page. FYI the SVGR processor adds 1em sizing to all icons so they can be re-sized using fontSize, although height and width is fine too. Looks like we missed applying a size to that particular SVG - your fix is fine.

**Comment by plittlewood-rpt** [comment]
Thanks - yes I caught the 1em thing when debugging it. Will merge this then. Thanks for checking over it

File: packages/ts-scripts/src/vite/base-vite.js (hunk 1 above)
Comment (plittlewood-rpt) [reply packages/ts-scripts/src/vite/base-vite.js:95]: I did indeed. I had to change this so that it would run in a windows WSL environment.

**Pull request merged**